ElastosBountyProgram/front-end/src/module/project/detail/Component.js

```diff
@@ -362,10 +362,16 @@ class C extends BaseComponent {
 
         // status checks
         if (detail.bidding && _.indexOf([TASK_STATUS.CREATED,
-                TASK_STATUS.PENDING], detail.status) < 0) {
+                TASK_STATUS.PENDING, TASK_STATUS.APPROVED], detail.status) < 0) {
             return ''
         }
 
+        let isApprovedBidding = false
+        if (detail.bidding && _.indexOf([TASK_STATUS.CREATED,
```

Comment:
Isnt that the same check as the if above? Can you go over that logic again and make sure it makes sense?